serious_stuff/lib/screens/HomePage.dart

```diff
@@ -21,6 +26,35 @@ class HomePage extends StatefulWidget {
 }
 
 class _MyHomePageState extends State<HomePage> {
+  Location location = new Location();
+  bool _serviceEnabled;
+  PermissionStatus _permissionGranted;
+  LocationData _locationData;
+
+  @override
+  void initState() {
+    super.initState();
+    _checkLocationPermission();
+  }
+
+  void _checkLocationPermission() async {
+    _serviceEnabled = await location.serviceEnabled();
+    if(!_serviceEnabled) {

```

Comment:
I believe we're going with spaces after control structure words?
```suggestion
    if (!_serviceEnabled) {
```